src_c/event.c

```diff
@@ -1269,6 +1269,18 @@ dict_from_event(SDL_Event *event)
             _pg_insobj(dict, "window", pgWindow);
             break;
         }
+        case SDL_DROPTEXT:
+        case SDL_DROPFILE:{
+            SDL_Window *window = SDL_GetWindowFromID(event->drop.windowID);
+            PyObject *pgWindow;
+            if (!window ||
+                !(pgWindow = SDL_GetWindowData(window, "pg_window"))) {
+                pgWindow = Py_None;
+            }
+            Py_INCREF(pgWindow);
+            _pg_insobj(dict, "window", pgWindow);
+            break;
+        }
```

Comment:
This chunk of code (minus the new `case`) is a duplicate of code like 2 lines above, you could just add those new cases above the existing piece instead of duplicating it.